### ISSUE TITLE
Accessibility problems tracking issue

### ISSUE THREAD

**Issue by roosmaa** [RiseVision/rise-react-wallet#48]
This is a tracking issue for various accessibility problems we still need to address.

- [x] ~Progressing to transaction overview and onwards is invisible to the screen reader~
- [x] The account overview isn't really usable with a screen reader
- [x] Use `<a href>` for navigation elements - #85 
- [x] Security tips are visible to screen reader before the button click
- [x] Settings pinned account button doesn't expose current value to accessibility tools
- [x] Migrate forms to dedicated dialogs with correct labelledby and describedby attributes
- [x] Modal dialog header buttons lack accessibility titles - #47
- [x] Verify mnemonics visual word placement indicator should be aria-hidden
- [x] AppBar comes before drawer content for accessibility tools when drawer is permanently open
- [x] AccountIcon should be aria-hidden to reduce the screen reader noise
